src/v3/src/transformer/phone/phoneEnrollmentTransformer.ts

```diff
@@ -76,14 +107,33 @@ export const transformPhoneEnrollment: IdxStepTransformer = ({ formBag, transact
     },
   };
 
-  const stepper: StepperLayout = {
-    type: UISchemaLayoutType.STEPPER,
-    elements: [
+  uischema.elements = [
+    titleElement,
+  ];
+
+  if (phoneMethodOptions.length === 1) {
+    const firstOptionMethod = phoneMethodOptions[0].value;
+    data['authenticator.methodType'] = firstOptionMethod;
+    if (firstOptionMethod === 'sms') {
+      uischema.elements = uischema.elements.concat([
+        smsInfoTextElement,
+        phoneNumberElement,
+        smsStepSubmitButton,
+      ]);
```

Comment:
Is `concat` necessary here any reason not to use `push` since `uischema.elements` has already been overwritten?

Reply:
Ah makes sense. Thx!